integration-tests/src/test/resources/apps/buildDeployWebAppInPod.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
copyright missing

Reply:
add

# Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
# Licensed under the Universal Permissive License v 1.0 as shown at
# http://oss.oracle.com/licenses/upl.